docs/src/components/Container/ComponentsContainer.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import { Heading, Tabs, Tab, Paragraph } from '@innovaccer/design-system';
+import "./Container.css"
 import { navigate } from 'gatsby';
+import { onScrollHandler } from './Helper';
 
 const ComponentsContainer = ({ children, pageTitle, relativePagePath, tabs, pageDescription, frontmatter }) => {
   const page = relativePagePath.split('/');
   const pageName = page[page.length - 1].split('.')[0];
   const isSiblingTab = relativePagePath.split('.')[0] === '/' + pageTitle.replace(/\s/g, '');
   const tabsList = isSiblingTab ? frontmatter?.tabs : tabs;
+  
+  React.useEffect(() => {
+    let element = document.getElementById('main-container');
```

Comment:
Please give some other id instead of `main-container`

Reply:
It's a predefined ID which is used here to detect the scroll event on the complete container page.